.github/workflows/publish_configs.yaml

```diff
@@ -0,0 +1,33 @@
+name: Publish config files
+on:
+  workflow_call:
+    inputs:
+      versions:
+        type: string
+        required: true
+    secrets:
+      S3_BUCKET_PATH:
+        required: true
+      AWS_ACCESS_KEY_ID:
+        required: true
+      AWS_SECRET_ACCESS_KEY:
+        required: true
+jobs:
+  build:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Check out repo
+        uses: actions/checkout@v3
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: eu-west-2
+      - name: Publish configs to artefacts S3 bucket
+        run: |
+          for version in $(echo ${{ inputs.versions }} | sed "s/,/ /g"); do
+            echo "Publishing version $version"
+            aws s3 cp psga/config/psga-schema-sars-cov-2.yaml ${{ secrets.S3_BUCKET_PATH }}/${version}/psga-schema-sars-cov-2.yaml
```

Comment:
I would like this to be dynamic rather than having to identify each one manually. Hard coded like this will do for a first version though

Reply:
It now lists over the files. I wasn't sure whether the configs folder will have all sorts of random things in it.